CONTRIBUTING.md

```diff
@@ -44,14 +44,20 @@ eval "$(pyenv init -)"
 eval "$(pyenv virtualenv-init -)"
 ```
 
-Then, create your virtual environment. While py42 runs on python 2.7 and 3.5+, a 3.6+ version is required for development in order to run all of the unit tests and style checks.
+Then, create your virtual environment.
 
 ```bash
 pyenv install 3.6.10
```

Comment:
Should we change these to 3.6.13?

Reply:
Good idea to update them while in there.